src/test/java/wooteco/subway/domain/SectionsTest.java

```diff
@@ -0,0 +1,212 @@
+package wooteco.subway.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.Assertions.tuple;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import wooteco.subway.exception.IllegalSectionException;
+
+class SectionsTest {
+
+    @DisplayName("새로 등록할 구간의 상행역과 하행역 중 노선에 이미 등록되어있는 역을 기준으로 새로운 구간을 추가한다.")
+    @Test
+    public void addNewSection() {
+        // given
+        List<Section> sectionList = new ArrayList<>();
+        sectionList.add(new Section(1L, 1L, 2L, 3L, 1));
+        final Sections sections = new Sections(sectionList);
+
+        // when & then
+        final Section section = new Section(2L, 1L, 1L, 2L, 1);
+        assertDoesNotThrow(() -> sections.add(section));
+    }
+    
+    @DisplayName("하나의 노선에는 갈래길이 허용되지 않기 때문에 새로운 구간이 추가되기 전에 갈래길이 생기지 않도록 기존 구간을 변경한다.")
+    @Test
+    public void forkRodeSameUpStation() {
+        // given
+        List<Section> sectionList = new ArrayList<>();
+        sectionList.add(new Section(1L, 1L, 1L, 3L, 7));
```

Comment:
```suggestion
        sectionList.add(new Section(1L, 1L, 3L, 7));
```

`Section`의 식별자는 이 맥락에선 불필요해 보이는데 제거해도 되지 않을까요? 비슷한 숫자들이 연달아 있다 보니 각 값의 의미를 이해하는 데 시간이 걸리는 것 같아요.

Reply:
<img width="507" alt="스크린샷 2022-05-14 오전 9 10 10" src="https://user-images.githubusercontent.com/57028386/168403064-8ab18969-fcd7-46d9-ab97-357ee1af6e23.png">
위와 같이 IDE의 도움을 받다 보니 고려해보지 못했던 부분이네요...😅
말씀해주신 것과 같이 해당 단위 테스트에서 Section의 식별자는 불필요하고, 이를 생략함으로써 코드를 읽는 사람으로 하여금 관심있는 부분만 집중하게 할 수 있을 것 같습니다! 수정해보겠습니다~_~

Reply:
나머지 부분에서도 Section의 식별자는 불필요하므로 제거하였습니다! 하지만 예외 적으로 `checkDistance()` 와 `sameSection()` 에서는 무엇에 의해서 예외가 발생하는지 분명히 해주고 싶어 Section의 id 값을 그대로 사용하도록 유지시켜 주었습니다!